spec/support/test_environment.rb

```diff
@@ -6,6 +6,7 @@
   config.include RspecTagMatchers
   config.include CustomMacros
   config.mock_with :rspec
+  config.deprecation_stream = '/dev/null'
```

Comment:
Ok. This still can be a surprise when someone wants to upgrade rspec. I'd not expect it to be silenced.